examples/enqueue_trial.py

```diff
@@ -0,0 +1,39 @@
+"""
+Optuna enqueue_trial example that optimizes a classifier configuration using sklearn.
+
+In this example, we optimize a classifier configuration for Iris dataset. We start a study with
+given parameter values, such as a default and a manually optimized.
+
+"""
+
+import sklearn.datasets
+import sklearn.model_selection
+import sklearn.svm
+
+import optuna
+
+
+def objective(trial):
+    iris = sklearn.datasets.load_iris()
+    x, y = iris.data, iris.target
+
+    svc_c = trial.suggest_float("svc_c", 1e-10, 1e10, log=True)
+    classifier_obj = sklearn.svm.SVC(C=svc_c, gamma="auto")
+
+    score = sklearn.model_selection.cross_val_score(classifier_obj, x, y, n_jobs=-1, cv=3)
+    accuracy = score.mean()
+    return accuracy
+
+
+if __name__ == "__main__":
+    study = optuna.create_study(direction="maximize")
+
+    # We enqueue a default parameter and a manually optimized parameter.
+    study.enqueue_trial({"svc_c": 1})
+    study.enqueue_trial({"svc_c": 10})
+
+    study.optimize(objective, n_trials=100)
+
+    print(study.trials[0])
+    print(study.trials[1])
+    print(study.best_trial)
```

Comment:
It may be a bit hard to compare the optimization results.
```console
FrozenTrial(number=0, value=0.9733333333333333, datetime_start=datetime.datetime(2020, 12, 2, 16, 4, 17, 127117), datetime_complete=datetime.datetime(2020, 12, 2, 16, 4, 24, 554043), params={'svc_c': 1}, distributions={'svc_c': LogUniformDistribution(high=10000000000.0, low=1e-10)}, user_attrs={}, system_attrs={'fixed_params': {'svc_c': 1}}, intermediate_values={}, trial_id=0, state=TrialState.COMPLETE)
FrozenTrial(number=1, value=0.98, datetime_start=datetime.datetime(2020, 12, 2, 16, 4, 17, 127421), datetime_complete=datetime.datetime(2020, 12, 2, 16, 4, 26, 445547), params={'svc_c': 10}, distributions={'svc_c': LogUniformDistribution(high=10000000000.0, low=1e-10)}, user_attrs={}, system_attrs={'fixed_params': {'svc_c': 10}}, intermediate_values={}, trial_id=1, state=TrialState.COMPLETE)
FrozenTrial(number=6, value=0.9866666666666667, datetime_start=datetime.datetime(2020, 12, 2, 16, 4, 26, 539575), datetime_complete=datetime.datetime(2020, 12, 2, 16, 4, 26, 555319), params={'svc_c': 4.732865944410664}, distributions={'svc_c': LogUniformDistribution(high=10000000000.0, low=1e-10)}, user_attrs={}, system_attrs={}, intermediate_values={}, trial_id=6, state=TrialState.COMPLETE)
```

How about extracting the parameter values and objective values as follows?
```console
C=1, Value=0.9733333333333333
C=10, Value=0.98
C=4.861487264595608, Value=0.9866666666666667
```